Assets/MixedRealityToolkit.Services/InputSystem/GazePointerVisibilityStateMachine.cs

```diff
@@ -65,7 +65,7 @@ private void UpdateStateEyeGaze(int numNearPointersActive, int numFarPointersAct
         {
             // If there are any far pointers active while eye gaze is valid, then
```

Comment:
Nit: Also update the comment here stating that it checks for both near and far

Reply:
Good catch! Will do ASAP.

Reply:
Whereeverrrrrrrrrrrrrrr you areeeeeeeeeeeeeeeeeeeeeeeeeeeeee

Reply:
Ehm?